.golangci.yml

```
@@ -0,0 +1,22 @@
+linters:
+  enable:
+    - errcheck
+    - errname
+    - errorlint
+    - goconst
+    - gofmt
+    - gofumpt
```

```suggestion
    - gofumpt
```

gofumpt supersede gofmt, so having both seems strange